go.mod

```diff
@@ -4,6 +4,11 @@ go 1.16
 
 require (
 	github.com/PaesslerAG/jsonpath v0.1.1
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.17.0
+	github.com/go-logr/logr v1.2.3
```

Comment:
Neat! No more dependency on terraform. We use zap in CNDS as well as part of controller-runtime